### PR TITLE
LPS-147124

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: [LPS-147124](https://issues.liferay.com/browse/LPS-147124)

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js [hunk 1]
```diff
@@ -172,8 +172,8 @@ function appendXMLAssignments(
 					);
 
 					if (
-						dataAssignments.autoCreate[index] !== null &&
-						dataAssignments.autoCreate[index] !== undefined
+						dataAssignments.autoCreate?.[index] !== null &&
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js:175]: @alinedoleron you can just replace the entire condition with `dataAssignments.autoCreate?.[index]` :)

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by naraandrade** [comment]
ci:stop

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**